src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1551,12 +1549,16 @@ public void updateCounts( final MoleculeCounts moleculeCounts,
             }
             return combinedSNVs;
         }
+
+        @VisibleForTesting ReadReport( final List<Interval> refCoverage, final List<SNV> snvList ) {
+            this.refCoverage = refCoverage;
+            this.snvList = snvList;
+        }
+
+        public static ReadReport NULL_REPORT = new ReadReport(new ArrayList<>(), new ArrayList<>());
     }
 
-    @VisibleForTesting static ReadReport getReadReport( final GATKRead read,
-                                                        final byte[] refSeq,
-                                                        final ReadCounts readCounts,
-                                                        final Reference reference ) {
+    @VisibleForTesting static ReadReport getReadReport( final GATKRead read ) {
```

Comment:
It would be nice if this function can be a ctor of `ReadReport`.
That is: the current single read ctor 
`public ReadReport( final GATKRead read, final Interval trim, final byte[] refSeq )` 
is only used in this function.

Reply:
I can certainly live with it, was just recommending.